### PR TITLE
new OTel config vars

### PR DESCRIPTION
**Pull request by vetinari**
PR for making `^observability_` accessible will follow...

### CODE REVIEW

File: cluster/manifests/01-admission-control/config.yaml [hunk 1]
```diff
@@ -26,6 +26,12 @@ data:
   pod.env-inject.variable._PLATFORM_OPENTRACING_LIGHTSTEP_COLLECTOR_PORT: "8443"
   pod.env-inject.variable._PLATFORM_OPENTRACING_LIGHTSTEP_COLLECTOR_HOST: "{{ .Cluster.ConfigItems.tracing_collector_host }}"
   pod.env-inject.variable._PLATFORM_OPENTRACING_LIGHTSTEP_ACCESS_TOKEN: "{{ .Cluster.ConfigItems.lightstep_token }}"
+  pod.env-inject.variable._PLATFORM_OBSERVABILITY_COLLECTOR_ENDPOINT: "{{ .Cluster.ConfigItems.observability_collector_endpoint }}"
+  pod.env-inject.variable._PLATFORM_OBSERVABILITY_COLLECTOR_PORT: "{{ .Cluster.ConfigItems.observability_collector_port }}"
+  pod.env-inject.variable._PLATFORM_OBSERVABILITY_COLLECTOR_SCHEME: "{{ .Cluster.ConfigItems.observability_collector_scheme }}"
+  pod.env-inject.variable._PLATFORM_OBSERVABILITY_METRICS_ENDPOINT: "{{ .Cluster.ConfigItems.observability_metrics_endpoint }}"
+  pod.env-inject.variable._PLATFORM_OBSERVABILITY_METRICS_PORT: "{{ .Cluster.ConfigItems.observability_metrics_port }}"
+  pod.env-inject.variable._PLATFORM_OBSERVABILITY_ACCESS_TOKEN: "{{ .Cluster.ConfigItems.lightstep_token }}"
```
Comment (zeitlinger) [inline cluster/manifests/01-admission-control/config.yaml:34]: also add `pod.env-inject.variable._PLATFORM_OBSERVABILITY_COMMON_ATTRIBUTE_CLOUD__ACCOUNT__ID : "{{ .Cluster.Alias }}"`

**Comment by vetinari** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Comment by gargravarr** [comment]
@vetinari please resolve the branch conflicts.

**Comment by vetinari** [comment]
:+1:

File: cluster/config-defaults.yaml [hunk 2]
```diff
@@ -702,4 +702,11 @@ deployment_service_cf_auto_expand_enabled: "false"
 deployment_service_enabled: "true"
 
 # Standard storageclass gp3 volume type
-storageclass_standard_gp3: "true"
+storageclass_standard_gp3: "false"
```
Comment (zeitlinger) [inline cluster/config-defaults.yaml]: you probably didn't want to change this line?

Comment (mikkeloscar) [reply cluster/config-defaults.yaml]: NO, don't change that :)

Comment (vetinari) [reply cluster/config-defaults.yaml]: oops, that happend during the merge :(

**Comment by mikkeloscar** [comment]
@vetinari This still has conflicts to `dev`

I'm fixing the build here which is unrelated: #5077

**Comment by vetinari** [comment]
fixed, didn't realize the other PR has been merged

**Comment by mikkeloscar** [comment]
@vetinari Please rebase once again to fix the build and then it should be good :)

**Comment by vetinari** [comment]
:+1:

**Comment by gargravarr** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**